doc/deploy.md

```diff
@@ -0,0 +1,69 @@
+# Deploy
+
+This document describes how to deploy Chaos Mesh for performing chaos experiments 
+on your application on Kubernetes. 
+
+## Prerequisites
+
+Before deploying Chaos Mesh, make sure the following items have been installed.
+
+- Kubernetes >= v1.12
+- [RBAC](https://kubernetes.io/docs/admin/authorization/rbac) enabled (optional)
+- [Helm](https://helm.sh/) version >= v2.8.2
+
+## Step 1: Get the Helm files
```

Comment:
Title could be confusing ?

Reply:
Good catch. Maybe "Get Chaos Mesh"? @cwen0